packages/react-server/core/ClientController.js

```diff
@@ -457,9 +457,10 @@ class ClientController extends EventEmitter {
 					}
 				}
 
-				if (element.containerOpen || element.containerClose){
-					return; // Nothing left to do.
-				}
+			}
+
+			if (element.containerOpen || element.containerClose){
+				return; // Nothing left to do.
```

Comment:
I think this broke with DOM preservation, though I'm not sure.  Don't want to hand container control objects off to React.